src/client/pages/ConnectPayment/components/TrustlyModal.tsx

```diff
@@ -53,7 +53,7 @@ export const TrustlyModal: React.FC<Props> = ({
     if (isSuccess) {
       onSuccess()
     }
-  }, [isSuccess])
+  }, [isSuccess, onSuccess])
```

Comment:
same here, i'm not sure `onSuccess` should be retrigged if that function changes. I don't think we should trust eslint blindly here

Reply:
Yeah I guess `onSuccess` is "new" on every render but since the initial state of `isSuccess` is false there won't be any calls to `onSuccess` on re-render right? Or am I thinking about it wrong? Anyways, `onSuccess` is never called when I run this locally since Trustly iframe is blocked 😕  But the tests for `TrustlyModal` are fine and onSuccess is only called once there.